From c53d4419269e916098238ef2bbd15ee2e9381cf0 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Sun, 3 Jul 2011 15:15:51 -0400
Subject: mmc: Add module.h to drivers/mmc users assuming implicit presence.

We are cleaning up the implicit presence of module.h; these guys are
some of the people who just assume it will be there.  Call it out
explitly for those that really need it.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
Signed-off-by: Chris Ball <cjb@laptop.org>
(cherry picked from commit 88b47679746b81534002bcba42da97ab82b5d12a)

Conflicts:

	drivers/mmc/card/mmc_test.c
	drivers/mmc/host/mmc_spi.c
	drivers/mmc/host/mxs-mmc.c
	drivers/mmc/host/sdhci-of-esdhc.c
	drivers/mmc/host/sdhci-of-hlwd.c
	drivers/mmc/host/sdhci-pci.c
	drivers/mmc/host/sdhci-pltfm.c
	drivers/mmc/host/sdhci-pxav2.c
	drivers/mmc/host/sdhci-pxav3.c
	drivers/mmc/host/sdhci-spear.c
	drivers/mmc/host/sdhci-tegra.c
	drivers/mmc/host/sdhci.c
	drivers/mmc/host/sdricoh_cs.c
	drivers/mmc/host/sh_mobile_sdhi.c
	drivers/mmc/host/tifm_sd.c
	drivers/mmc/host/via-sdmmc.c

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 drivers/mmc/host/sh_mmcif.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mmc/host/sh_mmcif.c b/drivers/mmc/host/sh_mmcif.c
index bd91c94..369366c 100644
--- a/drivers/mmc/host/sh_mmcif.c
+++ b/drivers/mmc/host/sh_mmcif.c
@@ -31,6 +31,7 @@
 #include <linux/platform_device.h>
 #include <linux/pm_runtime.h>
 #include <linux/spinlock.h>
+#include <linux/module.h>
 
 #define DRIVER_NAME	"sh_mmcif"
 #define DRIVER_VERSION	"2010-04-28"
-- 
1.7.10.2.565.gbd578b5

